cypress/integration/loginAngular.spec.js

```diff
@@ -0,0 +1,34 @@
+import { VAR, URL } from '../const/constants';
+import { IDs } from '../page-objects/loginAngular';
+import{ HAVETEXT } from '../const/helpers';
+
+context('Register', () => {
+  beforeEach(() => {
+    cy.visit(URL.index);
+    cy.get(IDs.idSingIn).click();
+  });
+
+  it('Sign up validate null', () => { 
+    cy.get(IDs.btnSingIn).should('not.have.class', 'disabled');
+
+   });
+
+  it('Sign up data incorrect', () => { 
+    
+    HAVETEXT(IDs.email,VAR.textSmall);
+    HAVETEXT(IDs.password,VAR.textSmall);
+    cy.get(IDs.btnSingIn).click();
+
+    cy.get(`.${IDs.errorRegister}`).contains(VAR.error);
```

Comment:
If we use constants to save the selectors, we should save the entire selector.
Example: the `IDs.errorRegister` shoud have `.error-messages`, not only `error-messages`.

Reply:
Change in the branch "NewArticule"